httpclient5/src/test/java/org/apache/hc/client5/http/utils/TestBase64.java

```diff
@@ -108,21 +108,6 @@ void decodeUnpadded() {
         checkDecode("BBBBAAA");
     }
 
-    @Test
-    void decodeUrlSafe() {
-        final char underscore = '_';
-        final char minus = '-';
-
-        checkDecode(fourOf(minus));
-        checkDecode(fourOf(underscore));
-    }
-
-    @Test
-    void mixedUrlAndRegularEncoded() {
-        checkDecode("++__");
-        checkDecode("--//");
-    }
-
```

Comment:
I don't even know how much it makes sense to test here. At most our wrapper, but not Base64 itself because it is not provided by us.

Reply:
Having dropped the special handling of URl-safe encoding, a lot less of this is needed. I'm cleaning it up.

Reply:
Tests have been cleaned up